enterprise/app/services/enterprise/billing/handle_stripe_event_service.rb

```diff
@@ -56,6 +57,10 @@ def change_plan_features
     account.save!
   end
 
+  def reset_captain_usage
+    account.reset_response_usage
```

Comment:
we will have to check if there is a plan change the happened in this event and reset the usage limits accordingly ?